### PR TITLE
Open source changes

### PR DESCRIPTION
**Pull request by voorepreethi**
This PR consists of 
1. License Attached
2. Security job attached to check the template security checks

### CODE REVIEW

**Comment by BuildWithJGang000** [comment]
.

**Comment by BuildWithJGang000** [comment]
bugs in the package end

**Review by sanathramesh — DISMISSED** [review]

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -23,4 +23,22 @@ jobs:
...
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Run CFN Nag Security Checks
+        uses: stelligent/cfn_nag@master
+        with:
+          input_path: .
+          extra_args: -o sarif
+          output_path: cfn_nag.sarif
+      - name: Upload CFN Nag SARIF
+        uses: github/codeql-action/upload-sarif@v3
```
Comment (hrai-nr) [inline .github/workflows/pull_request.yaml:41]: we are using @v4 for checkout, but @v3 for this. It is intended or can be changed to have same version everywhere ?

Comment (voorepreethi) [reply .github/workflows/pull_request.yaml:41]: That depends on Individual version release. 
latest version in checkout is v4v where as in upload serif is v3

**Review by BuildWithJGang000 — APPROVED** [review]

**Comment by BuildWithJGang000** [comment]
fixed ..

**Pull request merged**